package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quantumblack/kedro-viz",
-  "version": "6.3.2",
+  "version": "6.3.3",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
```

Comment:
I know the PR is merged. I was trying to build the package and saw the version here is not updated to 6.3.3
@rashidakanchwala should we do that ?

Reply:
@tynandebold  -- how do we do this best?